core_api/src/app.py

```diff
@@ -199,6 +230,22 @@ def get_file_chunks(file_uuid: UUID) -> list[Chunk]:
     return storage_handler.get_file_chunks(file_uuid)
 
 
+@app.get("/file/{file_uuid}/status", tags=["file"])
+def get_file_status(file_uuid: UUID) -> FileStatus:
+    """Get the status of a file
+
+    Args:
+        file_uuid (str): The UUID of the file to get the status of
+
+    Returns:
+        File: The file with the updated status
+    """
+
+    status = storage_handler.get_file_status(file_uuid)
```

Comment:
Can we add an `unknown` status here for files that don't exist to obfuscate that fact